responder/server/server.go

```diff
@@ -141,20 +141,20 @@ func (s *Server) startListener(ip net.IP, port int) {
 	// listen to incoming udp ntp.
 	conn, err := net.ListenUDP("udp", &net.UDPAddr{IP: ip, Port: port})
 	if err != nil {
-		log.Fatal(err)
+		log.Fatalf("listening error: %s", err)
 	}
 	defer conn.Close()
 
 	// Allow reading of hardware/kernel timestamps via socket
 	if err := ntp.EnableKernelTimestampsSocket(conn); err != nil {
-		log.Fatalln(err)
+		log.Fatalf("enabling timestamp error: %s", err)
 	}
 
 	for {
 		// read HW/kernel timestamp from incoming packet
 		request, nowHWtimestamp, returnaddr, err := ntp.ReadPacketWithKernelTimestamp(conn)
 		if err != nil {
-			log.Fatalln(err)
+			log.Errorf("read timestamp error: %s", err)
```

Comment:
Not sure about this one. Usually failure in reading means something is badly broken. With Errorf we will just go millions of problems without recovering.

Reply:
Why does it have the `continue` afterwards then?

In general I would expect the app to continue running even if it fails to read a packet/timestamp.

Reply:
Yes, fatalf and continue together were a clear mistake

Reply:
Also it's probably "read packet with timestamp error"